### PR TITLE
Add link to Solid World August recording

### PR DESCRIPTION
**Pull request by Mitzi-Laszlo**

### CODE REVIEW

File: _posts/newsletter/2020-08-06-this-month-in-solid.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ categories: [Updates]
 
 ### [Events](https://solidproject.org/events)
 
-Solid World is an opportunity to meet people who are interested in or currently working on Solid. If you missed Solid World you can [watch the recording here]().The [next Solid World](https://www.eventbrite.com/e/solid-world-tickets-115477011851?aff=erelexpmlt) will happen on the 3rd September 2020. 
+Solid World is an opportunity to meet people who are interested in or currently working on Solid. If you missed Solid World you can [watch the recording here](https://vimeo.com/446435972?utm_source=email&utm_medium=vimeo-cliptranscode-201504&utm_campaign=29220).The [next Solid World](https://www.eventbrite.com/e/solid-world-tickets-115477011851?aff=erelexpmlt) will happen on the 3rd September 2020.
```
Comment (Vinnl) [inline _posts/newsletter/2020-08-06-this-month-in-solid.md:14]: I'd remove the tracking code, unless you're explicitly using them to see how many people found it through solidproject.org?

```suggestion
Solid World is an opportunity to meet people who are interested in or currently working on Solid. If you missed Solid World you can [watch the recording here](https://vimeo.com/446435972).The [next Solid World](https://www.eventbrite.com/e/solid-world-tickets-115477011851) will happen on the 3rd September 2020. 
```

**Comment by RubenVerborgh** [comment]
Just to let you know: as per the revised process, it's also possible to make small edits like this one directly to the branch (in this case `staging`) without having to go through a PR. That might make things even smoother.

**Comment by Mitzi-Laszlo** [comment]
Also as a non-Creator non-Editor?

**Comment by RubenVerborgh** [comment]
> Also as a non-Creator non-Editor?

Oh no, just Creators. And others should not have permissions anyway if all is set up correctly.

**Pull request merged**